src/templating/templating.js

```diff
@@ -224,7 +224,7 @@
             if (typeof bindingValue == "string" || bindingValue['name']) {
                 // It's a named template - clear the element
                 ko.virtualElements.emptyNode(element);
-            } else if ('nodes' in bindingValue) {
+            } else if (bindingValue['nodes']) {
```

Comment:
Wanted to have this as a separate PR, Needed for a scenario where I wanted
to conditionally bind nodes.
2016. júl. 8. du. 4:36 ezt írta ("Brian M Hunt" <notifications@github.com>):

> In src/templating/templating.js
> <https://github.com/knockout/knockout/pull/1951#discussion_r70083532>:
>
> > @@ -224,7 +224,7 @@
> >              if (typeof bindingValue == "string" || bindingValue['name']) {
> >                  // It's a named template - clear the element
> >                  ko.virtualElements.emptyNode(element);
> > -            } else if ('nodes' in bindingValue) {
> > +            } else if (bindingValue['nodes']) {
>
> Why change this?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/knockout/knockout/pull/1951/files/ac0ca4090bb0dfff594124f5970e4c3f524123bf#r70083532>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AF0RgszX1Jn8Ran3MNZIqlTIAi6m5QkQks5qTmBLgaJpZM4GyvlW>
> .
>